nats-tools/nas/README.md

```diff
@@ -1,20 +1,18 @@
 # nats-account-server
 
-The [NATS Account Server](https://github.com/nats-io/nats-account-server) is an HTTP server that hosts and vends JWTs for nats-server 2.0 account authentication. The server supports an number of stores which enable it to serve JWTs from:
+The [NATS Account Server](https://github.com/nats-io/nats-account-server) is an HTTP server that hosts and vends [JWTs](../../nats-server/configuration/securing_nats/jwt/README.md) for nats-server 2.0 account authentication. 
+The server supports an number of stores which enable it to serve account [JWTs](../../nats-server/configuration/securing_nats/jwt/README.md) from:
 
-* a directory
-* an [NSC](../nsc/nsc.md) directory
-* memory \(for testing purposes\)
+* a [directory](nas_conf.md#directory-configuration)
+* an [NSC](../nsc/nsc.md) [directory](nas_conf.md#nsc-configuration)
 
-The server can operate in a _READ ONLY_ mode where it serves content from a directory, or in notification mode, where it can notify a NATS server that a JWT in the store has been modified, updating the NATS server with the updated JWT.
+> For testing purposes the nats server can be configured with a [memory resolver](../../nats-server/configuration/securing_nats/jwt/resolver.md#memory) that avoids usage of the account server.
```

Comment:
memory resolver can also be a valid option for prod, I'd leave out the `testing purposes` part